### PR TITLE
Change tab text of typography from "Styling strategies" to "Style strategies"

### LINKED ISSUE #3148: change TAB text from “Styling strategies” to “Style strategies”
https://next.carbondesignsystem.com/elements/typography/styling-strategies

### PR DESCRIPTION
**Pull request by abhilipsasahoo03**
Closes #3148 

**Change log**

- Changed tab text of typography section by replacing "Styling strategies" with "Style strategies"

- Updated corresponding links to "Style strategies" in the pages

- Improved minor verbiage

### CODE REVIEW

**Comment by abhilipsasahoo03** [comment]
I have read the DCO document and I hereby sign the DCO.

**Comment by abhilipsasahoo03** [comment]
@kingtraceyj @tay1orjones kindly look into this and let me know if any change is required.

**Review by aagonzales — APPROVED** [review]
This looks good but should we add a redirect incase other pages on the site were also linking to the old tab name page?

**Review by tay1orjones — CHANGES_REQUESTED** [review]
Good catch on the redirects - @abhilipsasahoo03 the redirect can be placed alongside these others https://github.com/carbon-design-system/carbon-website/blob/e0059b25c61478950b18258e4187465fd4771d79/gatsby-node.js#L64-L68

**Comment by abhilipsasahoo03** [comment]
> Good catch on the redirects - @abhilipsasahoo03 the redirect can be placed alongside these others https://github.com/carbon-design-system/carbon-website/blob/e0059b25c61478950b18258e4187465fd4771d79/gatsby-node.js#L64-L68

Hi @tay1orjones @aagonzales I have added the redirect...kindly review.

File: gatsby-node.js [hunk 1]
```diff
@@ -66,4 +66,9 @@ exports.createPages = ({ actions }) => {
     toPath: '/developing/vue-tutorial/overview',
     isPermanent: true,
   });
+    createRedirect({ 
+     fromPath: '/guidelines/typography/styling-strategies', 
+     toPath: '/guidelines/typography/style-strategies', 
+     isPermanent: true, 
+   });
```
Comment (tay1orjones) [inline gatsby-node.js:73]: Sorry just some minor formatting here that I can suggest and commit

```suggestion
  createRedirect({
    fromPath: '/guidelines/typography/styling-strategies',
    toPath: '/guidelines/typography/style-strategies', 
    isPermanent: true,
  });
```

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**